### ISSUE TITLE
works with react-native?

### ISSUE THREAD

**Issue by razorman8669** [skortchmark9/reselect-tools#7]
any idea how to get this to work (if possible) with react-native development?  I tried it out of the box to no avail.

**Comment by skortchmark9** [comment]
Hey @razorman8669 thanks for your question. I've only dabbled with react native so I'm not sure! The chrome extension works by injecting some js into the page and then calling a function on window.__RESELECT_TOOLS__. Does react native even use chrome dev tools? (I hope so)

That being said, it's possible to call the same functions yourself! You can output a selector graph as json with [selectorGraph](https://github.com/skortchmark9/reselect-tools#selectorgraphselectorkey--defaultselectorkey). You can then upload that selectorGraph into the [example app here](https://github.com/skortchmark9/reselect-tools/blob/master/examples/your-app.js).

Let me know if that helps!

**Comment by skortchmark9** [comment]
IIRC the react / redux devtools do some fun RPC stuff to make them work for react native, but that's not what we're doing here.

tl;dr I never tested it with react native and I doubt it works.
